web/load.js

```diff
@@ -21,7 +21,7 @@
         svg.setAttribute('id', id);
         svg.setAttribute('data', file);
         svg.setAttribute('type', 'image/svg+xml');
-        svg.setAttribute('style', 'visibility:hidden');
+        svg.setAttribute('style', 'visibility:hidden;position:absolute;left:-40px;');
```

Comment:
Is the `left:-40px` required, or `position:absolute` would be enough? I just don't like that we have some hardcoded number there. If it's required, what if we put `0px`?